systemtest/src/main/java/io/strimzi/systemtest/utils/specific/MetricsUtils.java

```diff
@@ -99,14 +100,27 @@ public static String getResourceMetricPattern(String kind, String namespace) {
     }
 
     public static void assertMetricResourcesHigherThanOrEqualTo(MetricsCollector collector, String kind, int expectedValue) {
+        Predicate<Double> higherOrEqualToExpected = actual -> actual >= expectedValue;
```

Comment:
Just curious...wouldn't be better to factor out this local Predicate to the class scope and also implement an equal Predicate? Therefore you can negate the condition of the following method (i.e., `assertMetricResourcesLowerThanOrEqualTo`) with an equal predicate. WDYT?

Reply:
they are checking 
>= 
and 
<= 

which are not exactly negations.

Reply:
as `=>` and `<=` are not exactly negations I would have to provide additional implementations of another predicate which would have to be in OR relation with negation of another one which may be confusing to read/mantain.